### PR TITLE
Bear: update to 3.0.19.

### PR DESCRIPTION
**Pull request by motorto**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/Bear/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/rizsotto/Bear"
 changelog="https://raw.githubusercontent.com/rizsotto/Bear/master/ChangeLog.md"
```
Comment (classabbyamp) [inline srcpkgs/Bear/template:13]: looks like the changelog was removed sometime between 2.4.4 and 3.0 :/

Comment (motorto) [reply srcpkgs/Bear/template:13]: You are right, sorry didn't check the links. 

Should I simply remove it ? Didn't find an alternative in the Bear git page

Comment (classabbyamp) [reply srcpkgs/Bear/template:13]: yeah, doesn't seem like they're doing a changelog anymore

**Comment by motorto** [comment]
Done, thanks for the input !

**Pull request merged**